apps/ui/tsconfig.json

```
@@ -6,7 +6,11 @@
 
 		/* Bundler mode */
 		"moduleResolution": "bundler",
-		"jsx": "react-jsx"
+		"jsx": "react-jsx",
+		"baseUrl": ".",
+		"paths": {
+			"@assets/*": ["./assets/*"]
```

to prevent confusion with a possible `@assets` npm organisation, it'd be best we stick to our usual `@/*` pattern
```suggestion
			"@/assets/*": ["./assets/*"]
```